pages/common/pixiecore.md

```diff
@@ -0,0 +1,8 @@
+# pixiecore
+
+> Pixiecore is a tool to manage network booting of machines.
```

Comment:
```suggestion
> A tool to manage network booting of machines.
```
@bl-ue what do you think

Don't commit this until others have responded.

Reply:
Yeah I kinda like that one because it doesn't duplicate "pixicore," but other than that they're equivalent and I don't have much of an opinion.

Reply:
if a user has entered tldr pixiecore they know what they have searched so its pointless to include it